tests/check-refs.sh

```diff
@@ -40,3 +40,7 @@ nix-build -o $RESULT check-refs.nix -A test7
 
 # test10 should succeed (no disallowed references).
 nix-build -o $RESULT check-refs.nix -A test10
+
+# test11 should succeed (no referenceable paths)
+test11=$(nix-build -o $RESULT check-refs.nix -A test11)
+! nix-store -q --references "$test11" | grep -q .
```

Comment:
```suggestion
[[ -z $(nix-store -q --references "$test11") ]]
```

might be a bit easier to understand